### PR TITLE
Fix padding bottom assistant builder

### PR DESCRIPTION
**Pull request by PopDaph**
## Description

Handling @flvndvd 's feedback from https://dust4ai.slack.com/archives/C050A0S2Z7F/p1709647456718279?thread_ts=1709646750.382989&cid=C050A0S2Z7F

Adding back a div that was disabled. 

## Risk

/

## Deploy Plan

/

### CODE REVIEW

File: front/components/assistant_builder/AssistantBuilder.tsx [hunk 1]
```diff
@@ -499,7 +499,7 @@ export default function AssistantBuilder({
           })()}
           <PrevNextButtons screen={screen} setScreen={setScreen} />
         </div>
-        {false && <div className="flex flex-col space-y-8 pb-16 pt-8"></div>}
+        <div className="flex flex-col space-y-8 pb-16 pt-8"></div>
```
Comment (philipperolet) [inline front/components/assistant_builder/AssistantBuilder.tsx:502]: why a div and not a spacing as class of the div above?
can have a look if needed :+1:

**Review by philipperolet — COMMENTED** [review]
quick question on this

File: front/components/assistant_builder/AssistantBuilder.tsx (hunk 1 above)
Comment (PopDaph) [reply front/components/assistant_builder/AssistantBuilder.tsx:502]: It was added here: https://github.com/dust-tt/dust/commit/091bd229af538cce9370d4ab7a1a872dfeb609bb

Comment (philipperolet) [reply front/components/assistant_builder/AssistantBuilder.tsx:502]: Ok will have a look later, you can merge in the meantime

**Review by philipperolet — APPROVED** [review]

File: front/components/assistant_builder/AssistantBuilder.tsx (hunk 1 above)
Comment (flvndvd) [reply front/components/assistant_builder/AssistantBuilder.tsx:502]: It should probably be part of the App Layout at some point 🙃.

**Pull request merged**